libs/datasets/timeseries.py

```diff
@@ -740,6 +740,8 @@ def _diff_preserving_first_value(series):
     first_date = cases.notna().idxmax()
     if pd.notna(first_date):
         new_cases[first_date] = cases[first_date]
+        new_cases[:first_date] = 0
```

Comment:
I'm surprised setting all days before the first day to zero doesn't cause more of a problem; I thought many cases timeseries start midway through the pandemic and the jump from 0 to that midpoint would cause a wacky Rt around that date. Could you check on the location with largest cases[first_date] that is modified by this change?

Reply:
It looks like this affects the counties in PR the most.  most others (with one or two exceptions) were < 10 on their first day.  I might change this back - it shouldn't affect rt calcs

Reply:
IIRC pandas slices are inclusive and so this will overwrite first_date?

Reply:
n/m. Just saw you removed this. 😁

Reply:
Oh good call about loc label indexing being inclusive at the end. I was curious about a fix... something like
```
first_date_ix = cases.notna().argmax()
if pd.notna(cases.iat[first_date_ix]:
    new_cases.iat[first_date_ix] = cases.iat[first_date_ix]
    new_cases.iloc[:first_date_ix] = 0
```

Moot point now.